src/helpers/utils.ts

```diff
@@ -47,6 +47,10 @@ export function formatISO8601Date(year: string | number, month: string | number,
     return null
 }
 
+export function dateFormat (dateString: string, locales?: Intl.LocalesArgument) {
+    return (new Date(dateString)).toLocaleDateString(locales, {timeZone: 'UTC'})
```

Comment:
@moust introduced a dependency to `date-fns`, it seems they have an [API](https://date-fns.org/v4.1.0/docs/format) for this kind of conversions ?

Reply:
Yep. `intlFormat(new Date(dateString), { timeZone: "UTC" }, { locale: locales })`

https://date-fns.org/v4.1.0/docs/intlFormat

/!\ Attention de ne pas utiliser la méthode `format` car j'ai volontairement exclus d'importer les assets de traduction. `intlFormat` propose la même API mais en reposant sur l'API native `Intl.DateTimeFormat`.